server/dialer/dial.go

```diff
@@ -84,12 +96,64 @@ func HandlePost(r *http.Request) map[string][]string {
 	return m
 }
 
+func handleFilePost(r *http.Request) (multipart.File, *multipart.FileHeader, error) {
+	if r.Method == "POST" {
+		r.ParseMultipartForm(30)
+		file, handler, err := r.FormFile("myfile")
```

Comment:
According to the go doc
> FormFile returns the first file for the provided form key. FormFile calls ParseMultipartForm and ParseForm if necessary.

That is you don't need to call ParseMultipartForm unless you want to restrict the upload size. 

For example 
```go
    // Parse our multipart form, 10 << 20 specifies a maximum
    // upload of 10 MB files.
    r.ParseMultipartForm(10 << 20)
```

Reply:
yea i need here to specify the maximum size of the file